src/controllers/executionPlanWebviewController.ts

```diff
@@ -0,0 +1,157 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+import { ReactWebViewPanelController } from "./reactWebviewController";
+import * as ep from "../reactviews/pages/ExecutionPlan/executionPlanInterfaces";
+import { WebviewRoute } from "../sharedInterfaces/webviewRoutes";
+import * as LocalizedConstants from "../constants/localizedConstants";
+import { homedir } from "os";
+import { fileExists } from "../utils/utils";
+
+export class ExecutionPlanWebViewController extends ReactWebViewPanelController<
+  ep.ExecutionPlanWebViewState,
+  ep.ExecutionPlanReducers
+> {
+  constructor(
+    context: vscode.ExtensionContext,
+    private executionPlanService: ep.ExecutionPlanService,
+    private executionPlanContents: string,
+    // needs ts-ignore because linter doesn't recognize that fileName is being used in the call to super
+    // @ts-ignore
+    private xmlPlanFileName: string
+  ) {
+    super(
+      context,
+      `${xmlPlanFileName} ${LocalizedConstants.executionPlan}`,  // Sets the webview title

```

Comment:
Can't you just force this as string instead of using ts-ignore?

Reply:
i don't think it was a problem of it being defined or not, it just straight didn't register that it was being used